cmd/skupper/skupper_podman_site.go

```diff
@@ -78,7 +79,26 @@ func (s *SkupperPodmanSite) Create(cmd *cobra.Command, args []string) error {
 	if routerCreateOpts.Ingress != types.IngressNoneString {
 		// Validating ingress hosts (required as certificates must have valid hosts)
 		if len(site.IngressHosts) == 0 {
-			return fmt.Errorf("At least one ingress host is required")
+			// Get all unicast IP addresses on the machine
+			addresses, err := net.InterfaceAddrs()
+			if err != nil {
+				return fmt.Errorf("Cannot get a default ingress host")
+			}
+			var ingressHosts []string
+			for _, address := range addresses {
+				ipnet, ok := address.(*net.IPNet)
+				if ok && !ipnet.IP.IsLoopback() && ipnet.IP.To4() != nil {
+					ipv4ValidAddress := ipnet.IP.String()
+					// Try a reverse lookup of a valid IPv4 address
+					fqdns, err := net.LookupAddr(ipv4ValidAddress)
+					if err != nil {
+						ingressHosts = append(ingressHosts, ipv4ValidAddress)
```

Comment:
... then you could get rid of this and just add the fqdns in case error is nil. Wdyt?

Reply:
Yes, LGTM.